process/block/preprocess/smartContractResults.go

```diff
@@ -518,6 +519,16 @@ func (scr *smartContractResults) ProcessMiniBlock(miniBlock *block.MiniBlock, ha
 
 		processedTxHashes = append(processedTxHashes, miniBlockTxHashes[index])
 
+		if scr.flagOptimizeGasUsedInCrossMiniBlocks.IsSet() {
+			//TODO: Remove the commented code if the meta-chain revert problem is fixed
```

Comment:
TODO still relevant?

Reply:
If the system test will pass the TODO and the commented code will be removed